apps/i18n/common/ko_kr.json

```diff
@@ -534,11 +534,11 @@
   "findLocalClassButton": "클래스 찾기",
   "finish": "마침",
   "forTeachersOnly": "선생님 전용",
-  "gdprDialogHeader": "Do you agree to using a website based in the United States?",
-  "gdprDialogDetails": "Data from your use of this site may be sent to and stored or processed in the United States.",
-  "gdprDialogSeePrivacyPolicy": "For details see our privacy policy",
-  "gdprDialogLogout": "Log out",
-  "gdprDialogYes": "예",
+  "gdprDialogHeader": "미국에 기반을 두고 있는 웹사이트 사용에 동의하십니까?",
+  "gdprDialogDetails": "귀하의 본 사이트 사용에 따른 데이터는 미국으로 전송되어 보관 또는 처리될 수 있습니다.",
+  "gdprDialogSeePrivacyPolicy": "(상세한 내용은 당사의 개인 정보 보호 정책을 참조하십시오.)",
+  "gdprDialogLogout": "로그 아웃",
+  "gdprDialogYes": "네",
```

Comment:
Here's one that was already translated. According to google translate it looks like a Yes/Yeah. I'll revert and take the existing volunteer translation.